src/Module/TokenLogin.php

```diff
@@ -166,7 +159,7 @@ protected function getResponse(Template $template, ModuleModel $model, Request $
         $template->action   = $request->getRequestUri();
         $template->slabel   = $this->translate('MSC.login');
         $template->value    = StringUtil::specialchars($this->authenticationUtils->getLastUsername());
-        $template->formId   = 'tl_login_' . $model->id;
+        $template->formId   = 't_login_' . $model->id;
```

Comment:
This is one of the main culprits I guess. We can change the form submit as we like.

```suggestion
        $template->formId   = 'tl_token_login_' . $model->id;
```

Reply:
> I only chose `t_login` to be consistent with your route🙂

Oh 🤔 that's nice